integrations/src/test/java/io/jenkins/plugins/casc/GiteaOrganisationFolderTest.java

```diff
@@ -0,0 +1,36 @@
+package io.jenkins.plugins.casc;
+
+import hudson.model.TopLevelItem;
+import io.jenkins.plugins.casc.misc.ConfiguredWithCode;
+import io.jenkins.plugins.casc.misc.JenkinsConfiguredWithCodeRule;
+import jenkins.branch.OrganizationFolder;
+import jenkins.model.Jenkins;
+import org.jenkinsci.plugin.gitea.GiteaSCMNavigator;
+import org.junit.Rule;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.assertTrue;
+
+public class GiteaOrganisationFolderTest {
```

Comment:
There's no point to this test as far as I can tell

Reply:
They do get linted AFAIK. Got errors during implementation. 😀
Do you mean they are not executed?

If not, how would I write a proper test for a JobDSL based configuration?

Reply:
see line 21 and 22, it's disabled, no point in adding this test imo. the other test is what I was after